index.html

```diff
@@ -0,0 +1,209 @@
+            <img src="./image/image2.png" alt="hello">
+        <h1>TONIC</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="./image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="./image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+    </div>
+
+        <div class="card2">
+            <img src="./image/card2.png" alt="some">
+        <h1>Multi-Post <br>Stories</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="./image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="./image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+         </div>
+
+
+        <div class="card3">
+            <img src="./image/card3.png" alt="">
+        <h1>TONIC</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="./image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="./image/image4.png" alt="count2"></li>
+            <li>2015</li>
+            
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+        </div>
+
+
+
+
+
+
+
+
+        <div class="card4">
+            <img src="./image/card4.png" alt="">
+        <h1>Multi-Post <br>Stories</h1> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="./image/image4.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="./image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+
+
+
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+        </div>
+    
+    </section>
+
+
+
+
+
+    <section>
+    
+    <div class="about">
+        <h1>About <br> MySelf</h1>
```

Comment:
Kindly, I suggest you only put an H1 heading, for other titles follow the sequence H2, H3... depending on the importance. this will greatly help the accessibility of your website.